libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -0,0 +1,318 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BrowserModule, By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { expect, expectAsync } from '@skyux-sdk/testing';
+import { SkyIdService } from '@skyux/core';
+import { SkyPopoverHarness } from '@skyux/popovers/testing';
+import {
+  SkyTheme,
+  SkyThemeMode,
+  SkyThemeService,
+  SkyThemeSettings,
+  SkyThemeSettingsChange,
+} from '@skyux/theme';
+
+import { BehaviorSubject } from 'rxjs';
+
+import { SkyHelpInlineModule } from '../help-inline/help-inline.module';
+
+import { HelpInlineTestComponent } from './fixtures/help-inline.component.fixture';
+
+describe('Help inline component', () => {
+  async function checkAriaPropertiesAndAccessibility(
+    ariaLabel: string,
+    ariaControls: string | null,
+    ariaExpanded: string | null,
+  ): Promise<void> {
+    const helpInlineElement =
+      fixture.nativeElement.querySelector('.sky-help-inline');
+    expect(helpInlineElement?.getAttribute('aria-label')).toBe(ariaLabel);
+    expect(helpInlineElement?.getAttribute('aria-controls')).toBe(ariaControls);
+    expect(helpInlineElement?.getAttribute('aria-expanded')).toBe(ariaExpanded);
+    await expectAsync(fixture.nativeElement).toBeAccessible();
+  }
+
+  async function getPopoverTestHarness(): Promise<{
+    popoverHarness: SkyPopoverHarness;
+  }> {
+    const loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+    const popoverHarness = await loader.getHarness(SkyPopoverHarness);
```

Comment:
Instead of interacting with the popover harness just to click on the button, can't you simply `.click()` on the help inline button?
```
function getHelpButton(fixture: ComponentFixture<HelpInlineTestComponent>): HTMLButtonElement {
  return fixture.nativeElement.querySelector('.sky-help-inline');
}

const helpButton = getHelpButton();
helpButton.click();
```